pkg/services/folder/model.go

```diff
@@ -0,0 +1,90 @@
+package folder
+
+import (
+	"time"
+)
+
+type Folder struct {
+	ID          int64
+	UID         string
```

Comment:
I think `ParentUID` is missing

Reply:
Also I think `URL`, `UpdatedBy` and `CreatedBy` do not apply here (as per the table schema). `HasACL` probably as well, but I'm not sure.

Reply:
`OrgID` on the other hand seems to be missing

Reply:
also I don't insist but it would be handy to include the `ParentUID`

Reply:
Yep thank you!